<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterEditor</class>
 <widget class="QWidget" name="ParameterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_sigma_apr">
           <property name="text">
            <string>a priori stdev (sigma)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_sigma_apr"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_conf_pr">
           <property name="text">
            <string>confidence probability</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_conf_pr"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_tol_abs">
           <property name="text">
            <string>tolerance abs. terms</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_tol_abs"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_sigma_act">
           <property name="text">
            <string>actual type of stdev used</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_sigma_act">
           <item>
            <property name="text">
             <string>a posteriori</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>a priori</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_update_cc">
           <property name="text">
            <string>update constrained coord.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBox_update_cc">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_axes_xy">
           <property name="text">
            <string>axes  x and y are oriented</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBox_axes_xy">
           <item>
            <property name="text">
             <string>ne</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sw</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>es</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>en</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nw</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>se</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ws</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_angles">
           <property name="text">
            <string>angles are observed</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBox_angles">
           <item>
            <property name="text">
             <string>clockwise</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>counter-clockwise</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_algorithm">
           <property name="text">
            <string>algorithm</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBox_algorithm">
           <item>
            <property name="text">
             <string>gso</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>svd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>cholesky</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>envelope</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_ang_units">
           <property name="text">
            <string>angular units</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="comboBox_ang_units">
           <item>
            <property name="text">
             <string>400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>360</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_adj_covband">
           <property name="text">
            <string>adjusted cxx band width</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_adj_covband"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_epoch">
           <property name="text">
            <string>epoch</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="lineEdit_epoch"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_latitude">
           <property name="text">
            <string>latitude</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="lineEdit_latitude"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_ellipsoid">
           <property name="text">
            <string>ellipsoid</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QComboBox" name="comboBox_ellipsoid"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
